server/routes/login.js

```diff
@@ -183,11 +183,7 @@ module.exports = () => (router) => {
 
         res.render('pages/two-factor-auth', { authError: false, csrfToken: res.locals.csrfToken })
       } else {
-        req.user.employeeName = await getStaffMemberEmployeeName(
-          userAuthentication.ApiUrl,
-          utilities.getStartMonth(),
-          req.user.token,
-        )
+        req.user.employeeName = jwtDecode(req.user.token).name
```

Comment:
Were we using the property `employeeName` on the last PR. Thought you named it something different. Not sure if that matters

Reply:
we were, employeename is the friendly name, not the ID